src/redux/dataReducer.js

```diff
@@ -0,0 +1,102 @@
+/* eslint-disable no-unused-vars, prefer-destructuring */
```

Comment:
- [ ] eslint are here to help us write clean code. Please don't disable them.

Reply:
Hello,

I tried to solve the issue about `prefer-destructuring` but I was not able to find another way of doing it.
If you may give a suggestion I'd appreciate it.